src/fetch/index.ts

```
@@ -5,39 +5,43 @@ import {
 
-type IsValidUrl<
+type CheckUrl<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   QueryDef extends Record<string, unknown> | undefined,
   Url extends string,
   Query extends string | undefined = ExtractQuery<Url>,
   QueryKeys extends string = Query extends string ? ToQueryUnion<Query> : never,
-> = IsValidQuery<
+> = CheckQuery<
   // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/ban-types
   QueryDef extends Record<string, any> ? QueryDef : {},
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)